pipelines/build/js/ci.yml

```diff
@@ -62,6 +62,19 @@ parameters:
     displayName: 'Pre-release version to publish.'
 
 steps:
+  # Validate change files
+  - ${{ if and( eq(parameters.WithBeachball, 'true'), eq(variables['Build.Reason'], 'PullRequest') ) }}:
+      - pwsh: |
+          $targetBranch = "$(System.PullRequest.TargetBranch)".replace('refs/heads/', 'origin/')
+          Write-Host "##vso[task.setVariable variable=TargetBranch]$targetBranch"
+          Write-Host "Resolved target branch: $targetBranch"
+        displayName: 'Resolve target branch'
+
+      - script: |
+          yarn beachball check -b $(TargetBranch)
+        displayName: 'Validate change files'
+        workingDirectory: ${{ parameters.ProjectRoot }}
```

Comment:
`yarn install` won't have happened before the `build` template i think, so this probably won't work 😟

Reply:
That figures! Too bad.. Let's call this one off then.